### DISCUSSION TITLE
username squatting question

### DISCUSSION THREAD

**Discussion by ibx34** [community/community#31048] [Other Feature Feedback, Questions, & Ideas]
I found an account that has been inactive for ~8 years, and I would like the username. Despite the policy outlined [here](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy), I can't send a support ticket or report the user.

If I can report it, why are there no methods of reporting, it?
If I can't report it, why do the docs say that I can?

GitHub docs say:

**GitHub prohibits account name squatting, and account names may not be reserved or inactively held for future use. Accounts violating this name squatting policy may be removed or renamed without notice. Attempts to sell, buy, or solicit other forms of payment in exchange for account names are prohibited and may result in permanent account suspension.**

No other discussions have given a clear answer or have linked to a github.community discussion which only redirects to this home page.

**Comment by airtower-luna** [comment]
GitHub staff have confirmed ([for example here](https://github.com/community/community/discussions/18486#discussioncomment-2949969)) that they don't release usernames on request. The policy only says that name squatting and account buying/selling can lead to permanent suspension, not that the names will become available to others.

**Reply by ssakib4050** [reply]
It's really sad because I want a name and the user just registered a username for like 6 years ago & after that account is totally inactive. and github messaging feature is also removed.

**Reply by landsman** [reply]
yeah, I'm sad as well ... what a shame
